db/migrate/20200930085956_create_osbbs.rb

```diff
@@ -0,0 +1,17 @@
+class CreateOsbbs < ActiveRecord::Migration[6.0]
+  def change
+    create_table :osbbs do |t|
+      t.string :name
```

Comment:
think of which field should have db indexes
as we will have search for OSBB later

Reply:
Resolve